Hannah-Sten/TeXiFy-IDEA
# Find usages for user-defined commands

I probably broke quite some things while doing this, so please check whatever you can think of and when possible write tests for it.

- [x] Go to declaration for user-defined commands
- [x] Find usages for user-defined commands
- [x] Also take \renewcommand into account
- [x] Go to declaration for labels
~~Find usages for labels~~
- [x] Go to declaration for files
- [x] Go to declaration for \includegraphics

#### Summary of additions and changes

* Add find usages provider
* Create command definition references correctly
* Translate many helper functions from the LatexPsiImplUtil to Kotlin

#### How to test this pull request

* Ctrl+B around for commands defined with \newcommand or \renewcommand
* Check go to declaration for labels and files

#### Wiki


- [x] Updated the wiki: https://github.com/Hannah-Sten/TeXiFy-IDEA/wiki/Find-usages
